### ISSUE TITLE
Livesplit crash on Edit Layout Cancel

### ISSUE THREAD

**Issue by Gyoo** [Gyoo/LiveSplit.SplitsBet#26]
Issue encountered on commit [a95081e](https://github.com/Gyoo/LiveSplit.SplitsBet/commit/a95081e7a7d7f44bd096ae6a21e94da70a7b7cb2)

Title is self explaining, Livesplit is "crashing due to an unknown reason" when I press cancel on the Edit Layout screen. The settings for SplitsBet seem to work correctly but don't seem to be saved or loaded correctly. These 2 things should be related.

**Comment by 0x0ade** [comment]
What does the Windows event log say? (I'm just about to test it via Wine by myself... T.T=;)

**Comment by Gyoo** [comment]
LiveSplit.exe Error: 0 : String hasn't been recognized as a valid Boolean value

   at System.Boolean.Parse(String value)
   at LiveSplit.SplitsBet.Settings.SetSettings(XmlNode settings) dans c:\Users\gyoze_000\Documents\Visual Studio 2013\Projects\LiveSplit.SplitsBet\LiveSplit.SplitsBet\Settings.cs:line 57
   at LiveSplit.SplitsBet.SplitsBetComponent.SetSettings(XmlNode settings) in c:\Users\gyoze_000\Documents\Visual Studio 2013\Projects\LiveSplit.SplitsBet\LiveSplit.SplitsBet\SplitsBetComponent.cs:line 469

It doesn't say exactly where Livesplit crashes but my opinion is that it comes from there

**Comment by 0x0ade** [comment]
It crashes upon setting UseGlobalTime from the XmlNode, which means - just as I said in my comment on [commit a95081e7a7d7f44bd096ae6a21e94da70a7b7cb2](https://github.com/Gyoo/LiveSplit.SplitsBet/commit/a95081e7a7d7f44bd096ae6a21e94da70a7b7cb2#diff-5de131ebdf7e27234c9e864583d96857R51) - SplitsBet crashes due to a backwards compatibility issue.

Do you prefer System.Xml's serializer (most stable / tested; can cause incompatibilities with previous settings and problems with specific classes / types), FEZMod's XmlHelper (uses some workarounds and manual deserialization of specific objects, but it works unexpectedly well) or simply fix it manually?

**Comment by Gyoo** [comment]
Maybe @CryZe can fix it tomorrow. I tried to manage SetSettings and GetSettings from what he showed me so I assume it should work but I might be doing something wrong here.

tl;dr manual fix should do the work.

**Comment by Gyoo** [comment]
I fixed it. The problem was that some data weren't bind correctly here and there + variables needed default values.
